galsim/config/util.py

```diff
@@ -248,6 +249,23 @@ def recursive_copy(d):
 
     return config1
 
+class SafeManager(BaseManager):
+    """There are a few places we need a Manager object.  This one uses the 'fork' context,
+    rather than whatever the default is on your system (which may be fork or may be spawn).
+
+    Starting in python 3.8, the spawn context started becoming more popular.  It's supposed to
+    be safer, but it wants to pickle a lot of things that aren't picklable, so it doesn't work.
+    Using this as the base class instead, should work.  And doing it it one place means that we
+    only have one place to change this is there is a different strategy that works better.
+    """
+    def __init__(self):
+        if sys.version_info >= (3,8):
```

Comment:
Can we just use (3,0)?  That'd make it more obvious to simplify in the future when we eventually drop support for py2.

Reply:
(and same comment for most of the (3,8) checks...)

Reply:
Yes.  Good point.